opencti-platform/opencti-graphql/src/modules/ingestion/ingestion-csv-domain.ts

```diff
@@ -43,18 +49,52 @@ export const findCsvMapperForIngestionById = (context: AuthContext, user: AuthUs
   return storeLoadById<BasicStoreEntityCsvMapper>(context, user, csvMapperId, ENTITY_TYPE_CSV_MAPPER);
 };
 
+export const createOnTheFlyUser = async (context: AuthContext, user: AuthUser, input: IngestionCsvAddInput) => {
+  const groups = await findDefaultIngestionGroup(context);
+  if (groups.length === 0) {
+    throw FunctionalError('You have not defined a default group for ingestion users', {});
+  }
+  const { platform_organization } = await getEntityFromCache<BasicStoreSettings>(context, SYSTEM_USER, ENTITY_TYPE_SETTINGS);
+  const newlyCreatedUser = await addUser(context, context.user, {
```

Comment:
```suggestion
  const newlyCreatedUser = await addUser(context, user, {
```
Do not use user from context, there is some use case to use another user than the one that is in context.